### ISSUE TITLE
[BUG] flyteadmin image is not available in registry

### ISSUE THREAD

**Issue by evalsocket** [flyteorg/flyte#2671]
### Describe the bug

We've installed the flyte-core helm chart in version 1.0.2 on one of our clusters yesterday, and it came up successfully. Whilst trying to upgrade another cluster today, we did get an ImagePullBackoff when trying to pull [cr.flyte.org/flyteorg/flyteadmin-release:v1.0.2](http://cr.flyte.org/flyteorg/flyteadmin-release:v1.0.2).

I can also confirm not being able to pull that image locally:
```
❯ docker pull cr.flyte.org/flyteorg/flyteadmin-release:v1.0.2
Error response from daemon: manifest unknown
```

Has anything changed here recently?

### Expected behavior

NA

### Are you sure this issue hasn't been raised already?

- [X] Yes

### Have you read the Code of Conduct?

- [X] Yes

**Comment by evalsocket** [comment]
cc: @EngHabu

**Comment by EngHabu** [comment]
We have included an important security fix in the [flyte v1.1.0 release](https://github.com/flyteorg/flyte/releases/tag/v1.1.0). We highly encourage everyone to upgrade. A CVE will be released on July 12th that will include more information about the vulnerability. If you have specific questions, please don’t hesitate to DM me until then.
